ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -422,19 +423,21 @@ Check Launched Workbench Is The Correct One
 Get Openshift Notebook CR From Workbench
     [Documentation]    Retrieves name of Notebook CR corresponding to a workbench
     [Arguments]     ${workbench_title}      ${namespace}
-    ${rc}  ${cr_name}=    Run And Return Rc And Output   oc get notebook -n ${namespace} -o jsonpath='{.items[?(@.metadata.annotations.openshift\\.io/display-name=="${workbench_title}")].metadata.name}'
-    RETURN    ${rc}    ${cr_name}
+    ${result}=    Run Process
```

Comment:
why? was it causing issues?

Reply:
you could check if ${rc} is different from 0 then it fails. I don't see an actual need of changing it, but ok if you have a stronger opinion.

Reply:
It happened few times when that kw was called, for example look at `Delete Workbench From CLI`:
```
${_}  ${cr_name}=    Get Openshift Notebook CR From Workbench
    ...    workbench_title=${workbench_title}  namespace=${ns_name}
IF    "${cr_name}" == "${EMPTY}"    BREAK
```
In the old way, when `Get Openshift Notebook CR From Workbench` failed, then `${cr_name}` was not empty, but included the stderr, which then did not lead to the `BREAK` of the WHILE. 
Using `Run Process` in `Get Openshift Notebook CR From Workbench`, and any other KW out there would not lead to such failure, since we return the stdout only, and in our case empty value, which leads to the BREAK as expected.